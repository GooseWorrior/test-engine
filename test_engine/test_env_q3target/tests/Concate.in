ab

ab
a
abc
q
abc
a
abc
q
(ab)c
abc
a
q
a(bc)
abc
a
q
((ab)c)d
abcd
a
q

