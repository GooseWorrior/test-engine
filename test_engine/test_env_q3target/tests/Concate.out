Concatenation (a,b)
false
true
false
false
Concatenation (a,Concatenation (b,c))
false
true
Concatenation (Concatenation (a,b),c)
true
false
Concatenation (a,Concatenation (b,c))
true
false
Concatenation (Concatenation (Concatenation (a,b),c),d)
true
false

